arangod/Aql/OptimizerRules.cpp

```diff
@@ -2916,7 +2916,7 @@ void arangodb::aql::scatterInClusterRule(Optimizer* opt,
     plan->registerNode(gatherNode);
     TRI_ASSERT(remoteNode);
     gatherNode->addDependency(remoteNode);
-    if (!elements.empty() && gatherNode->collection()->numberOfShards() > 1) {
+    if (!elements.empty() && gatherNode->collection()->numberOfShards() != 1) {
```

Comment:
@mchacki I suppose a comment for this could be helpful

Reply:
Yes agreed, added